command/agent/agent.go

```diff
@@ -281,8 +283,6 @@ func (c *cmd) run(args []string) int {
 			sig = s
 		case <-service_os.Shutdown_Channel():
 			sig = os.Interrupt
-		case <-c.shutdownCh:
-			sig = os.Interrupt
```

Comment:
This was removed because it did nothing (the channel was never closed), and we already have a case for handling signals.